File: Rock_Paper_Scissor.c
#include <stdlib.h>
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <time.h>
typedef long long ll;
int gran(){
    srand(time(NULL));
    return rand()%3;
}
int greater(char c1, char c2){
    if ((c1=='r' && c2=='s') || (c1=='p' && c2=='s') || (c1=='s' && c2=='p'))
    {
        return 1;
    }
    else if (c1==c2){
        return -1;
    }
    else {
        return 0;
    }
}
int main(int argc, char const *argv[])
{
    int scorep=0,temp,t2;
    int scorec=0;
    char playerchar, compchar;
    char dict[]={'r','p','s'};
    printf("Welcome to the game");
    printf("Choose 1 for rock 2 for paper 3 for scissor\n");
    for (int i = 0; i < 3; i++)
    {
        scanf("%d",&temp);
        playerchar=dict[temp-1];
        printf("You choose %d\n",temp);
        printf("You choose %c\n",playerchar);

        printf("Computers turn\n");
        t2=gran()+1;
        printf("%d\n",t2);
        compchar=dict[t2-1];
        printf("Computer choose %c\n",compchar);


        if (greater(compchar,playerchar)==1)
        {
            scorec++;
        }
        else  if (greater(compchar,playerchar)==-1)
        {
            scorep++;
        }
        else
        {
            scorep++;
        }
        
    }
    if (scorep > scorec)
    {
        printf("You win");
    }
    else if (scorep < scorec)
    {
        printf("You Loose");
    }
    else
    {
        printf("its a draw");
    }
    
    return 0;
}

File: 100Projects_from_easy_to_hard/Day5/Rock_Paper_scissor.py
import random
a = int(input("What do you choose? Type 0 for Rock, 1 for Paper or 2 for Scissors.\n"))
x = random.randint(0,2)
l = ["    _______\n---'   ____)\n      (_____)\n      (_____)\n      (____)\n---.__(___)\n\n","    _______\n---'   ____)____\n          ______)\n          _______)\n         _______)\n---.__________)\n\n","    _______\n---'   ____)____\n          ______)\n       __________)\n      (____)\n---.__(___)\n\n"]
if a == 0:
    print(l[a])
    if x == 0:
        print("Computer chose:\n")
        print(l[x])
        print("It's a draw")
    elif x == 1:
        print("Computer chose:\n")
        print(l[x])
        print("You lose")
    else : 
        print("Computer chose:\n")
        print(l[x])
        print("You win!")
elif a== 1:
    print(l[a])
    if x == 0:
        print("Computer chose:\n")
        print(l[x])
        print("You win!")
    elif x == 1:
        print("Computer chose:\n")
        print(l[x])
        print("It's a draw")
    else : 
        print("Computer chose:\n")
        print(l[x])
        print("You lose")
else:
    print(l[a])
    if x == 0:
        print("Computer chose:\n")
        print(l[x])
        print("You lose")
    elif x == 1:
        print("Computer chose:\n")
        print(l[x])
        print("You win!")
    else : 
        print("Computer chose:\n")
        print(l[x])
        print("It's a draw")


